cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)

```

Comment:
suggest you use `2^n - 1` length

Reply:
I didn't quite understand what you meant. But I corrected it to 127